### ISSUE TITLE
add prediction for test.

### ISSUE THREAD

**Issue by Anshul22Verma** [Anshul22Verma/MIDAS#1]
Need to add predict function for a dir with all the test-images arranged similar to the training set. 

1. Define generators and predict on them and return the accuracy along with actual predictions and true labels for any-other analysis that the user wants to make. 🙂
2. A predict variation that takes in X_test and Y_test like MNIST test set is already defined and is called *analyze_x_y* possibly use analyze for the other predict logic.

**Comment by Anshul22Verma** [comment]
functions for predictions have been added in commit https://github.com/Anshul22Verma/MIDAS/commit/c8cf84c62b2ae13b88b0ad52b240736837453767 :

1. prediction for an image.
2. prediction for data in training format.
3. prediction for all the test data in one folder. (kaggle-format)
